.travis.yml

```diff
@@ -8,7 +8,7 @@ services:
   - mysql
 
 addons:
-  postgresql: "9.3"
+  postgresql: "9.4"
```

Comment:
I've clarified this in a separate commit 6ff6d176123544751cd507299da2a57edef41224